src/WordList.php

```
@@ -65,6 +65,10 @@ public static function fromArray(array $words): self
             throw WordListException::empty();
         }
 
+        if (count($words) < 2) {
+            throw WordListException::insufficientEntropy();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\NicoBleiler\Passphrase\Exceptions\WordListException' in method 'fromArray'. (undefined)

(StaticAccess)